app/models/maths_and_physics/school_eligibility.rb

```diff
@@ -60,7 +65,8 @@ def eligible_current_school?
     private
 
     def eligible_local_authority_district?
-      ELIGIBLE_LOCAL_AUTHORITY_DISTRICT_CODES.include?(@school.local_authority_district.code)
+      EXPLICITLY_ELIGIBLE_SCHOOL_URNS.include?(@school.urn) ||
```

Comment:
Would it be better for this to be in a method of its own, like `explicity_eligible_school?`. Just avoids polluting this method with edge cases. You could then change the `eligible_current_school?` method to look a bit like this:

```ruby
def eligible_current_school?
  @school.open? &&
    (eligible_local_authority_district? || explicity_eligible_school?) &&
    (@school.state_funded? || @school.secure_unit?) &&
    @school.secondary_or_equivalent?
end
```

I know it complicates that method a little bit more, but it makes the intention clearer, for me anyhoo

Reply:
Yeah, I think it's a good idea. I'll change it.

Reply:
Another quick thought, it might be worth renaming this to something like `school_considered_in_eligible_district`, since we're not actually overriding any other eligibility tests.

Reply:
Just pushed a change, what do you think?

Reply:
Looks sane to me.